packages/main/src/CalendarHeader.ts

```diff
@@ -127,10 +127,12 @@ class CalendarHeader extends UI5Element {
 
 	onPrevButtonClick(e: MouseEvent) {
 		this.fireEvent("previous-press", e);
+		e.preventDefault();
```

Comment:
Hi @hinzzx it does not clear why we need this change, how it's related to the issue that is being fix, what was wrong, etc. Could add some info here. Thanks.

Reply:
As discussed offline - this change won't be needed anymore, and will be reverted !

Reply:
Discussed offline.